xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexMarketDataServiceRaw.java

```diff
@@ -132,4 +133,13 @@ public BitfinexLendDepth getBitfinexLendBook(String currency, int limitBids, int
       throw new BitfinexException(e.getHttpBody());
     }
   }
+
+  public List<Status> getStatus(List<CurrencyPair> pairs) throws IOException {
+    try {
+      return bitfinexV2.getStatus(
+          "deriv", BitfinexAdapters.adaptCurrencyPairsToTickersParam(pairs));
+    } catch (HttpStatusIOException e) {
```

Comment:
if you have a proper BitfinexExceptionV2 then this should not get thrown
please see my comment on BitfinexExceptionV2